### DISCUSSION TITLE
student pack benefits

### DISCUSSION THREAD

**Discussion by coder123-dv** [community/community#143818] [GitHub Education]
### Hi everyone,

How much time it will take to get my student pack benefits 🤔 my request got approved but showing you will receive your benefits soon.

**Comment by Parker936** [comment]
It typically takes up to 7 days for the benefits to be fully activated after your GitHub Student Pack request is approved. However, in many cases, it happens sooner. If it’s been more than a week and you still haven’t received the benefits, consider reaching out to [GitHub Education Support](https://support.github.com/contact?tags=education) for further assistance.

In the meantime, while waiting for your GitHub benefits, why not explore some new fashion inspirations? Check out [Carlanisa](https://carlanisa.com/) for beautifully designed Baju Kurung, Instant shawls, hijabs, and more!

**Comment by the-matrixneo** [comment]
For mine it took around 2 day .

**Comment by davevad93** [comment]
Hi @coder123-dv ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @coder123-dv was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
